runtime/src/bank/fee_distribution.rs

```diff
@@ -80,13 +80,25 @@ impl Bank {
         fee_details: &CollectorFeeDetails,
     ) -> (u64, u64) {
```

Comment:
out of scope:

haven't always been a huge fan of these tuples - could be confusing which is which?
for something as important as fees, would it make sense to have a simple struct here with named fields so it's less likely we mix them up?

Reply:
c0ebf47b297b465ec44c1a8d1d3c0ee21c4c2203

Reply:
c1aff58e2e339fc9ca3f12035544c0ec6cf68419

Reply:
Sorry, can we move those to a separate PR? the duplication/move of the code from fee rate governor to bank, is its own change. and the type change is another 😄

Reply:
yea sure, made a forced push

Reply:
#5399 to replace tuple w struct